exchanges/orderbook/depth_test.go

```diff
@@ -326,40 +326,81 @@ func TestWait(t *testing.T) {
 	wg.Add(100)
 	for x := 0; x < 100; x++ {
 		go func() {
-			<-wait.Wait(nil)
+			w := wait.Wait(nil)
+			wg.Done()
+			<-w
 			wg.Done()
 		}()
 	}
```

Comment:
While it took me awhile to try to understand how these tests work, I think it would be good to verify the wait values in all instances to verify that they were closed in the fashion you expected

Reply:
Will do